src/t9p.c

```diff
@@ -2633,7 +2584,7 @@ t9p_rename(t9p_context_t* c, t9p_handle_t dir, t9p_handle_t h, const char* newna
   }
 
   /* Update the file path for the node */
-  (void)t9p__string_release(h->str);
+  h->str = t9p__string_release(h->str);
```

Comment:
I'm guessing this was changed to resolve a warning, `h->str` is immediately reassigned

Reply:
this is how `t9p__string_release` is intended to be used, hence the `nodiscard` on it. So I just changed it to that to resolve a warning